user/admin.py

```diff
@@ -1,5 +1,26 @@
 from django.contrib import admin
-from django.contrib.auth.admin import UserAdmin
-from .models import User
+from django.contrib.auth.admin import UserAdmin as DjangoUserAdmin
+from django.utils.translation import gettext as _
+from user.models import User
 
-admin.site.register(User, UserAdmin)
+
+@admin.register(User)
+class UserAdmin(DjangoUserAdmin):
+    """Define admin model for custom User model with no email field."""
```

Comment:
The docstring states 'with no email field', but the email field is included in the fieldsets. Consider updating the docstring to accurately reflect the fields included in the admin model.